### PR TITLE
front : fix margin card width

### LINKED ISSUE #5565: Margin card has not the correct width in STDCM
See 

Image

Expected:

`osrd-config-item-container` does not have the same width than the other cards, should be the same...

Version: `8ecea1c4`

### PR DESCRIPTION
**Pull request by RomainValls**
closes #5565

### CODE REVIEW

File: front/src/modules/stdcmAllowances/components/STDCMAllowances.tsx [hunk 1]
```diff
@@ -49,7 +49,7 @@ const STDCMAllowances = () => {
   };
 
   return (
-    <div className="row osrd-config-item mb-2 osrd-config-item-container">
+    <div className="d-flex my-2 p-2 align-items-center osrd-config-item-container">
```
Comment (Yohh) [inline front/src/modules/stdcmAllowances/components/STDCMAllowances.tsx]:
```suggestion
    <div className="d-flex mb-2 align-items-center osrd-config-item-container">
```
padding seems useless here, as the full vertical margin

you could also change the vertical margin the same way in RunningTime.tsx

**Review by Yohh — CHANGES_REQUESTED** [review]

**Review by Yohh — APPROVED** [review]
LGTM :muscle:

File: front/src/modules/stdcmAllowances/components/STDCMAllowances.tsx [hunk 2]
```diff
@@ -49,7 +49,7 @@ const STDCMAllowances = () => {
   };
 
   return (
-    <div className="row osrd-config-item mb-2 osrd-config-item-container">
+    <div className="d-flex mb-2 align-items-center osrd-config-item-container">
```
Comment (clarani) [inline front/src/modules/stdcmAllowances/components/STDCMAllowances.tsx]: I'm not sure align-items-center is useful here, I removed it and it didn't change anything. What do you think ?
```suggestion
    <div className="d-flex mb-2 osrd-config-item-container">
```

**Review by clarani — CHANGES_REQUESTED** [review]
Left 1 small comment.
Can you squash your commits and rebase ? Thanks ! :)

**Comment by RomainValls** [comment]
C'est fait, merci !

**Review by clarani — APPROVED** [review]
LGTM thanks !! tested :white_check_mark:

**Pull request merged**